packages/layout/src/components/Sidebar.tsx

```diff
@@ -12,11 +12,11 @@ const Sidebar: React.FC<Properties> = ({ children, customStyle }) => {
 
   return (
     <div
-      className={open ? "sidebar-menu open" : "sidebar-menu"}
+      className={`sidebar-menu ${open ? "open" : "close"}`}
```

Comment:
Do we need `close` class? We can apply that style to `sidebar-menu` itself

Reply:
this is some thing oliver asked to keep. for easy over write.